src/terrain/clients/data_info/raw.clj

```diff
@@ -107,8 +107,8 @@
 
 (defn read-chunk
   "Uses the data-info read-chunk endpoint."
-  [user path-uuid position chunk-size]
-  (request :get ["data" path-uuid "chunks"]
+  [user path position chunk-size]
+  (request :get ["data" "by-path" "chunks" (url/url-encode path)]
```

Comment:
Not sure if this is the right thing to do, but I noticed this starting to fail while testing out some refactors (also fails in master) so I switched the endpoint.  I noticed both sonora and the DE are sending paths, so I assumed this would be ok.

Reply:
That's fine to do since I merged cyverse-de/data-info#44 earlier, but I'm concerned if the UUID versions are failing now. What are you seeing as far as that?

Reply:
... I see my mistake, I'll fix that. But this change is still fine, I just don't want to break the old one either, lol

Reply:
Ah ok, even better - I'm guessing I don't even need this change anymore.  Thanks!  I only tried testing opening a file in sonora and the DE and saw those 2 cases were using paths, but there might be cases where we are using UUIDs instead. 🤔

Reply:
We'd want to double-check nothing is trying to use a UUID version (I guess it'd have to be using the raw client directly), but ultimately what you're doing here is the plan, in order to avoid the UUID overhead -- the "uuid-for-path" you're doing here is another data-info call, and then if a UUID is passed to data-info it has to turn it back into a path anyway. So I think you're good either way.

Reply:
Ah ok, I removed this change for this PR since the old endpoint started working and I didn't do _that_ much research into what might still be using the UUID version 😄